documentation/release_notes.markdown

```diff
@@ -4,6 +4,43 @@ title: "Puppet Server: Release Notes"
 canonical: "/puppetserver/latest/release_notes.html"
 ---
 
+## Puppet Server 1.1.2
+
+Released October 19, 2015.
+
+This is a security and bug fix release in the Puppet Server 1.1 series; no new
+features have been added since 1.1.0. We recommend that all users upgrade.
+
+### Bug Fixes
+
+#### Make Certificate Authority and Master Private Keys Inaccessible to "World" Users
+
+Previous versions of Puppet Server would not explicitly set file permissions for certificate authority (CA) and Master private keys, which could leave both keys' readable by "world" users. Puppet Server 2.2 resolves this bug by automatically setting and enforcing a permissions change that limits read access to the Puppet Server user and group.
+
+This fix was also applied to [Puppet Server 2.1.x](#puppet-server-212) and [1.2.x](#puppet-server-122).
```

Comment:
@gguillotte Sorry for the belated comment here, but the references in this sentence don't look right to me.  We don't have plans yet for a 1.2.x release, so I think that part of this should be removed.  (We will include this fix in the 2.2.0 release later, but I'm not sure if you want to mention that here yet?)

Also, the link for `#puppet-server-212` seems like it might not be right.  When I click on it from within http://docs.puppetlabs.com/puppetserver/1.1/release_notes.html#puppet-server-112, my browser tries to go to http://docs.puppetlabs.com/puppetserver/1.1/release_notes.html#puppet-server-212, which doesn't exist.  Instead, I think it should be going to http://docs.puppetlabs.com/puppetserver/2.1/release_notes.html#puppet-server-212.

Since I merged this already, we'd need to have another PR put up to address this.  Let me know if you want one of us to do this or if you have it covered.  Thanks!